### PR TITLE
Improves #163: Removed Hard coded Dimensions

### PR DESCRIPTION
**Pull request by madhavrathi**
Fixes issue #204 

Changes: removed hard coded dimensions

### CODE REVIEW

File: app/src/main/java/vn/mbm/phimp/me/Camera2.java [hunk 1]
```diff
@@ -306,6 +307,9 @@ public void onClick(View v) {
 		camera_switch = (ImageButton)view.findViewById(R.id.switch_camera);
 		camera_switch.setImageResource(R.drawable.camera_switch);
 		buttonClick.setImageResource(R.drawable.takepic);
+        LinearLayout.LayoutParams parmsswitch = new LinearLayout.LayoutParams(Utils.getScreenWidth(getContext())/15,Utils.getScreenHeight(getContext())/15);
+        camera_switch.setLayoutParams(parmsswitch);
+        parmsswitch.setMarginStart(20);
```
Comment (Abhi2424shek) [inline app/src/main/java/vn/mbm/phimp/me/Camera2.java]: Please properly intend the code.

**Comment by phimpme-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Message">
          1 Message
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">:tada: Thanks for the PR @madhavrathi we hope you have changed the variables in the upload_apk_forks.sh its for easy moderation.</td>
    </tr>
  </tbody>
</table>

:link: Maintainers: the apk to test can be found at https://github.com/madhavrathi/phimpme-android/blob/apk/app-debug.apk

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

File: app/src/main/java/vn/mbm/phimp/me/Camera2.java [hunk 2]
```diff
@@ -355,6 +359,9 @@ public void onClick(View v) {
 		linear.bringToFront();
 		flash = (ImageButton)view.findViewById(R.id.flash);
 		flash.setImageResource(R.drawable.flash_on);
+                LinearLayout.LayoutParams parmsFlash = new LinearLayout.LayoutParams(Utils.getScreenWidth(getContext())/15,Utils.getScreenHeight(getContext())/15);
```
Comment (Abhi2424shek) [inline app/src/main/java/vn/mbm/phimp/me/Camera2.java]: There's a single unwanted whitespace

**Comment by madhavrathi** [comment]
@Abhi2424shek Please review

**Comment by madhavrathi** [comment]
@Abhi2424shek I just checked that the file was having indentations in both tabs and space(in different chunks of code), so i made it uniform by intending by spaces.
Please review

**Pull request closed without merging**